lib/heroku/command/pg_backups.rb

```diff
@@ -380,16 +384,16 @@ def restore_backup
                  backups.find { |b| transfer_name(b) == backup_name }
                end
       if backups.empty?
-        abort("No backups. Capture one with `heroku pg:backups capture`.")
+        abort("No backups for #{backup_app}. Capture one with `heroku pg:backups capture`.")
       elsif backup.nil?
-        abort("Backup #{backup_name} not found.")
+        abort("Backup #{backup_name} not found for #{backup_app}.")
       elsif !backup[:succeeded]
-        abort("Backup #{backup_name} did not complete successfully; cannot restore it.")
+        abort("Backup #{backup_name} for #{backup_app} did not complete successfully; cannot restore it.")
       end
       restore_url = backup[:to_url]
```

Comment:
This url should be fine if we retrieve it from another app? Or will we need to resolve it to a public url ?